bot/exts/info/tags.py

```diff
@@ -362,8 +292,8 @@ def accessible_tags_in_group(self, group: str, user: discord.Member) -> list[str
             if identifier.group == group and tag.accessible_by(user)
         )
 
-    @tags_group.command(name="get", aliases=("show", "g"), usage="[tag_group] [tag_name]")
-    async def get_command(self, ctx: Context, *, argument_string: Optional[str]) -> bool:
+    @tag_group.command(name="get")
```

Comment:
Is there a reason for removing the aliases ? Or aren't they allowed in app commands?

Reply:
The reason for using aliases was for ease of user but now as slash commands display all the commands it would kind of get clustered seeing one command take up all the space in the suggestions pop up, like eg: just writing `t` will take 6(3 for get, 3 for list) space. Also when just typing the user can see the commands and can now, even autofill the subgroup easily. And this is the reason discord has removed support for aliases for slash commands.